src/style.css

```diff
@@ -1 +1,56 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
```

Comment:
```suggestion
  justify-content: space-between;
```

Reply:
flex-direction: row; is redundant. It's default value